parser/ast.go

```diff
@@ -24,6 +24,7 @@ type NodeProgram struct {
 type NodeSelect struct {
 	WildCard bool
 	Count    bool
+	Distinct []string
```

Comment:
I might be wrong (code review at midnight 😅 ) but it seems that you are setting a new slice but it could be a boolean.

Reply:
This was a SQL interpretation failure :P I though we required `distinct` per field but it turns out we don't we this can be a bool